tasks/config-cluster.yml

```diff
@@ -1,21 +1,36 @@
 ---
 # Configure RabbitMQ for cluster
 
+- name: ensure hostnames are in the host file
+  lineinfile:
+    dest: /etc/hosts
+    line: "{{ hostvars[item]['ansible_default_ipv4']['address'] }} {{ hostvars[item]['ansible_hostname'] }}"
+  with_items: "{{ groups['rabbit'] }}" 
```

Comment:
What if I do not have a hosts group called 'rabbit'? (For example when I want to install two RabbitMQ clusters)